packages/dialtone-vue2/recipes/conversation_view/message_input/message_input.vue

```diff
@@ -797,20 +815,27 @@ export default {
     margin-right: var(--dt-space-500);
   }
 
+  &__button {
+    max-height: 2.8rem;
+    max-width: 2.8rem;
```

Comment:
Use tokens please

Reply:
Do we have a token for `2.8rem`? 24px is too small. 36px is too big. Current button is 34px which isn't a standard size.

Reply:
yeah there's no token for it, just use rem. You could also calc two tokens together, but I'm kind of of the opinion we should just use rem in these one off cases.